lib/github-glue.ts

```diff
@@ -91,6 +91,32 @@ export class GitHubGlue {
         return checks.data.id;
     }
 
+    /**
+     * Add a cc to a Pull Request
+     *
+     * @param {string} pullRequestURL the Pull Request to comment on
+     * @param {string} cc to add
+     * @returns the comment ID and the URL to the comment
+     */
+    public async addPRcc(pullRequestURL: string, cc: string):
```

Comment:
Technically, I would write this as `Cc` (i.e. with an upper-case `C` to indicate that this is a new CamelCased "word"). Not sure whether you want to change it? I'd be fine with leaving it as-is, but I figured I'd mention that I stumbled over it.

Reply:
Updating.